examples/hybrid_quantum_algorithms/Quantum_Monte_Carlo_Chemistry/afqmc/classical_afqmc.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env
```

Comment:
do we need this here, given that it's a file containing helper functions and is never run directly?